spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>spring-cloud-gcp-starters</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-gcp-starter-sql</artifactId>
+
+	<dependencies>
+		<dependency>
```

Comment:
M-m-m. Why do you never read links I point?..
That Boot reference says clearly:
> You can easily generate your own configuration meta-data 

We need this for the compile time to generate particular JSON data in the META-INF of our jar.
When end-user uses our jar as dependency, the IDE is able to read that metadata and allow to auto-complete `application.properties`.